deploy.cmd

```diff
@@ -0,0 +1,5 @@
+@echo off
```

Comment:
As the regular build uses .NET tool this probably should too here too.

.NET Core 2.1.3 was released on Azure app service a couple of days ago so should be possible.